BEPs/BEP-336.md

```diff
@@ -0,0 +1,501 @@
+<pre>
+  BEP: 336
+  Title: Shard Blob Transactions
```

Comment:
refer: https://github.com/bnb-chain/BEPs/pull/345/files
title could be: Implement EIP-4844: Shard Blob Transactions